src/core/textures/VideoBaseTexture.js

```diff
@@ -67,7 +67,7 @@ export default class VideoBaseTexture extends BaseTexture
          * @member {boolean}
          * @default true
          */
-        this.autoPlay = true;
+        this.autoPlay = source.autoplay;
```

Comment:
Substituting `source.autoplay !== false` would be non-breaking (default to `true`).

Reply:
am not understand sorry can you elaborate?
by default the **source.autoplay** it alway false;
the problem is that pixi forces the autoplay, without take the source parameter.
this dont allow to force video to stop.
but if we use the video source, we are allow to force the autoplay if we need.

Reply:
Could add a new constructor param that defaults to true, but allows you to override this value.

Reply:
@englercj yes we can also add a new arg in `fromVideo(video, scaleMode, sourceOptions)`
and also verify argument length in **VideoBaseTexture** to add options for the video source.
Options will be a obj `option:{autoplay:false} `parameter based on the **HTMLMediaElement**.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement

Using the `arguments[]`, it would be possible to avoid any breakChange.